mirage-entropy.opam

```diff
@@ -8,36 +8,22 @@ author:       ["Hannes Mehnert" "David Kaloper" "Anil Madhavapeddy" "Dave Scott"
 maintainer:   "david@numm.org"
 license:      "BSD2"
 
-build: [ "ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%" "--tests" "false"
-          "--with-mirage-xen" "%{mirage-xen:installed}%"
-          "--with-mirage-solo5" "%{mirage-solo5:installed}%"
-          "--with-ocaml-freestanding" "%{ocaml-freestanding:installed}%"
-]
+build: [ "dune" "build" "-p" "mirage-entropy"]
```

Comment:
I recommend using the "standard" dune-opam snippet with `subst` and `runtest` (even if there are no tests for now)

Reply:
fixed in f5201a0b48f8d4f2949d7c50364885282ae94bf4